src/main/kotlin/org/rust/cargo/commands/Cargo.kt

```diff
@@ -53,6 +57,8 @@ class Cargo(
 
     private val metadataCommandline: GeneralCommandLine get() = generalCommand("metadata", emptyList())
 
+    private fun rustfmtCommandline(filePath: String) = generalCommand("fmt", ParametersListUtil.parse("-v -- --write-mode=overwrite --skip-children $filePath"))
```

Comment:
It's much better to use `withParameters` here instead of `ParameterListUtils.parse` hack. What if `filename` includes some spaces?